# A custom message type for sending commands to a Franka gripper
# Define the message fields
float64 width
float64 speed
float64 force
float64 tolerance
